Python/pywarpx/picmi.py

```diff
@@ -1886,13 +1886,17 @@ def init(self, kw):
         self.absolute_tolerance = kw.pop("warpx_absolute_tolerance", None)
         self.self_fields_verbosity = kw.pop("warpx_self_fields_verbosity", None)
         self.magnetostatic = kw.pop("warpx_magnetostatic", False)
+        self.cfl = kw.pop("warpx_cfl", None)
 
     def solver_initialize_inputs(self):
         # Open BC means FieldBoundaryType::Open for electrostatic sims, rather than perfectly-matched layer
         BC_map["open"] = "open"
 
         self.grid.grid_initialize_inputs()
 
+        if self.cfl is not None:
```

Comment:
This check `self.cfl is not None` is not needed. When the parameters are processed, there is already a check, so if `pywarpx.warpx.cfl` is None, it is ignored.

Reply:
Thanks, fixed.